tailwind.config.js

```diff
@@ -11,8 +11,8 @@ export default {
         light: "#F3F4F6", // Light background color
       },
       height: {
-        header: "80px", // Custom height for the header
-        footer: "60px", // Custom height for the footer
+        header: "30px", // Custom height for the header
```

Comment:
yaha yrr 100vh-(x) aur header and footer ki values alag alag hai iska kal dekhte hai

Reply:
also a scrollbar is been added because of disperancy in these values :
![image](https://github.com/user-attachments/assets/8b730ecf-366e-4f62-8201-df5a361d6419)